src/python_server/server.py

```diff
@@ -3,35 +3,75 @@
 import sys
 import os
 import capnp
+import argparse
 
 script_dir=os.path.dirname(os.path.realpath(__file__))
 sys.path.insert(0, os.path.join(script_dir, "../"))
 
 from capnp_serial import messages_capnp
 from capnp_serial import mapping_capnp
 
-port = "5556"
-if len(sys.argv) > 1:
-    port =  sys.argv[1]
+## Stuttgart folks implement this function ##
+def compute_util_threshold(drop_ratio):
+    return 0.5 # TODO
 
-context = zmq.Context()
-socket = context.socket(zmq.REP)
-socket.bind("tcp://*:%s" % port)
+## Stuttgart folks implement this function ##
+def compute_utility(features):
+    return 0.6 # TODO
 
-while True:
-    #  Wait for next request from client
-    message = socket.recv()
-    util_msg  = messages_capnp.UtilityMessage.from_bytes(message)
+def handle_util_threshold_req(util_threshold_req):
```

Comment:
One last thing, let's move this functions inside a class, so later if they need to persist some state across calls it is not implemented as a global variable :)